test/nodes/test_llm_streaming.py

```
@@ -0,0 +1,616 @@
+
+        chat_fn.assert_called_once_with('test', stream=True)
+        assert answer.getText() == 'Hello world'
+
+        # Verify SSE call sequence: stream_start, token*3, stream_end
+        types = [c.args[0] for c in mock_sse.call_args_list]
+        assert types[0] == 'stream_start'
+        assert types[-1] == 'stream_end'
+        assert types.count('token') == 3
+
+    def test_empty_chunks_produce_empty_answer(self):
+        inst, mock_sse = _make_instance()
```

Ruff (0.15.7)

[warning] Unpacked variable `mock_sse` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)